man/rgbasm.5

```diff
@@ -324,9 +343,41 @@ with a zero constant as either operand will be constant 0, and
 .Sq ||
 with a non-zero constant as either operand will be constant 1, even if the other operand is non-constant.
 .Pp
-.Ic \&!
+.Sq \&!
 returns 1 if the operand was 0, and 0 otherwise.
 Even a non-constant operand with any non-zero bits will return 0.
+.Ss Integer functions
+Besides operators, there are also some functions which have more specialized uses.
+.EQ
+delim $$
+.EN
+.Bl -column "BITWIDTH(n)"
+.It Sy Name Ta Sy Operation
+.It Fn HIGH n Ta Returns the top 8 bits of
+.Ar n .
```

Comment:
Actually, it's the top 8 bits *if it was 16-bit*, but our numbers are 32-bit.

```suggestion
.It Fn HIGH n Ta Returns the top 8 bits of Ar n
after truncating it to 16 bits.
```
, maybe?

Reply:
How about "LOW returns n & $FF", "HIGH returns (n >> 8) & $FF"? Avoids confusion over "top/bottom" or "high/low", and whether to say "8 bits" or "byte".

Reply:
"Bits 8-15 of the value" could work...

Reply:
That still isn't clear that those bit are shifted down 8 places. (Assuming we even need to be pedantically clear here; "the top+bottom 8 bits" hasn't been getting ??confused?? questions so far.)

Reply:
> (Assuming we even need to be pedantically clear here; "the top+bottom 8 bits" hasn't been getting ??confused?? questions so far.)

Arguably, because it's rare for people to work with values outside of 16-bit range.

Perhaps “bits 8–15 of the value as a single byte” would be clear enough that they are shifted down. (Plus, hey—if people haven't been confused by it thus far, it's probably okay to just say “bits 8–15 of the value”, which is more correct while being no less vague than before.)

Reply:
Alternatively, document both as “convenience shortcut for {expr}”. Clear and concise, and would also clear [this](https://github.com/gbdev/rgbds/pull/1450#discussion_r1706933715) up.